helmchart/templates/service/service.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.service.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ include "chart.fullname" . }}-service"
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+spec:
+  type: ClusterIP
+  ports:
+    - port: 80
+      targetPort: 3000
+      protocol: TCP
+      name: http
+    - port: 35729
```

Comment:
I think this port is only used in the `app` component. 

There are other ports used by service btw, but I'm not sure which ones need to be exposed